performance_manager/lib/s3_utils.py

```diff
@@ -10,17 +10,20 @@
 
 
 def read_parquet(
-    filename: str,
+    filename: Union[str, List[str]],
     filters: Optional[Union[List[Tuple], List[List[Tuple]]]] = None,
 ) -> pandas.core.frame.DataFrame:
     """
-    read a parquet file from s3 and return it as a pandas dataframe
+    read parquet file or files from s3 and return it as a pandas dataframe
```

Comment:
`ParquetDataset` has the option to ingest list of files and return of table of all those object concatenated. Updated the `read_parquet` function to utilize.